app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -304,38 +317,29 @@ function AcuantCapture(
 
   /**
    * Calls onChange with next value and resets any errors which may be present.
-   *
-   * @param {Blob|string|null} nextValue Next value.
-   * @param {ImageAnalyticsPayload=} metadata Capture metadata.
    */
-  function onChangeAndResetError(nextValue, metadata) {
+  function onChangeAndResetError(
+    nextValue: Blob | string | null,
+    metadata?: ImageAnalyticsPayload | undefined,
```

Comment:
Do we need both `?:` and `| undefined` ?

```suggestion
    metadata?: ImageAnalyticsPayload,
```